Example/ConsentViewController_ExampleTests/LocalStorage/GDPRUserDefaultsSpec.swift

```diff
@@ -32,16 +32,17 @@ class GDPRUserDefaultsSpec: QuickSpec {
                 }
 
                 it("gets its value from the local storage") {
-                    localStorage.storage = [SPUserDefaults.IAB_CMP_SDK_ID_KEY: 99]
+                    let iabKey = "\(SPUserDefaults.IAB_KEY_PREFIX)_key"
```

Comment:
Does it make sense to move `_key` to be part of a `SPUserDefaults.IAB_KEY_PREFIX` ?

Reply:
I see it's being reused at least twice here

Reply:
Not really. I used `SPUserDefaults.IAB_KEY_PREFIX` to test a generic "iab" key (a piece of data belonging to the TCData). But the goal was to simply remove the specific `SPUserDefaults.IAB_CMP_SDK_ID_KEY` key, which was not used in the SDK.